tests/pthread/test_pthread_atexit.c

```diff
@@ -0,0 +1,42 @@
+#include <assert.h>
+#include <pthread.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "emscripten.h"
+#include "emscripten/threading.h"
+
+static _Atomic int threadCounter = 0;
+static _Atomic int running = 1;
+static pthread_t thread;
```

Comment:
We should probably be consistent and either drop the `static` here or add it to all he functions below.

Reply:
I dropped those `static` variables with commit 786473b83591d39792dcb197bc8add00692d57a7.